src/wp-admin/includes/template.php

```diff
@@ -402,10 +402,10 @@ function get_inline_data( $post ) {
  *
  * @global WP_List_Table $wp_list_table
  *
- * @param int    $position
- * @param bool   $checkbox
- * @param string $mode
- * @param bool   $table_row
+ * @param int    $position  Optional. Value to assign to the position input field.
+ * @param bool   $checkbox  Optional. Value to assign to the checkbox input field.
+ * @param string $mode      Optional. Use WP_Post_Comments_List_Table instead WP_Comments_List_Table.
+ * @param bool   $table_row Optional. Use table instead div.
```

Comment:
```suggestion
 * @param int    $position  Optional. The value of the 'position' input field. Default 1.
 * @param bool   $checkbox  Optional. The value of the 'checkbox' input field. Default false.
 * @param string $mode      Optional. If set to 'single', will use WP_Post_Comments_List_Table, otherwise WP_Comments_List_Table. Default 'single'.
 * @param bool   $table_row Optional. Whether to use a table instead of a div element. Default true.
```